rcsb/utils/dictionary/DictMethodEntryHelper.py

```diff
@@ -875,10 +876,16 @@ def addEntryInfo(self, dataContainer, catName, **kwargs):
             # INSTANCE FEATURES
             #
             ##
-            repModelL = ["1"]
-            if self.__commonU.hasMethodNMR(methodL):
-                repModelL = self.__getRepresentativeModels(dataContainer)
-            logger.debug("Representative model list %r", repModelL)
+            #repModelL = ["1"]
```

Comment:
Just remember to remove/clean-up this and other commented out old code prior to finalizing the merge